test/lit/passes/inlining_enable-tail-call.wast

```diff
@@ -568,26 +802,118 @@
   (i32.const 42)
  )
 )
+
 (module
  ;; CHECK:      (type $0 (func))
 
  ;; CHECK:      (func $0
  ;; CHECK-NEXT:  (local $0 i32)
  ;; CHECK-NEXT:  (block $__inlined_func$1
  ;; CHECK-NEXT:   (block
+ ;; CHECK-NEXT:    (block $__return_call
+ ;; CHECK-NEXT:     (block
+ ;; CHECK-NEXT:      (br $__return_call)
```

Comment:
The lack of `local.set` here demonstrates that we successfully skipped using a local for this parameter. The test output still has a local because of the next inlining iteration.

Reply:
Got it, thanks. Yeah, that `local.set` has two causes in these tests now made this hard to see...